rllib/agents/mock.py

```diff
@@ -7,7 +7,7 @@
 from ray.rllib.utils.deprecation import deprecation_warning
 
 deprecation_warning(
-    old="ray.rllib.agents.callbacks",
```

Comment:
Just out of curiosity, why was this here before and why can we now change it to "mock"?

Reply:
probably a copy & paste error.
this is the mock file, not callbacks.